### PR TITLE
#653 add samples for listAssertions of api-fluent

### PR DESCRIPTION
**Pull request by BimuratMukhtar**
______________________________________
I confirm that I have read the [Contributor Agreements v1.0](https://github.com/robstoll/atrium/blob/master/.github/Contributor%20Agreements%20v1.0.txt), agree to be bound on them and confirm that my contribution is compliant.

### CODE REVIEW

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import ch.tutteli.atrium.api.fluent.en_GB.get
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class ListAssertionSamples {
+
+    // subject under test
+    private val sut = listOf(1, 2, 3)
```
Comment (robstoll) [inline apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt:11]: Have a look at the rebiew for pairAssertions the same applies here

**Review by robstoll — CHANGES_REQUESTED** [review]

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt (hunk 1 above)
Comment (BimuratMukhtar) [reply apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt:11]: ok

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+import ch.tutteli.atrium.api.fluent.en_GB.get
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class ListAssertionSamples {
+
+    @Test
+    fun get() {
```
Comment (robstoll) [inline apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt:11]: Please use again the same naming schema, getFeature here and `get` where one can pass an assertionCreator-lambda

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        val list = listOf(0, 1, 2)
+
+        expect(list).get(0).toBe(0)
+
+        fails {
+            // index is out of bound
```
Comment (robstoll) [inline apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt:17]: Nice 👍

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            expect(list).get(3).toBe(3)
```
Comment (robstoll) [inline apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt:18]: Please explain,as in pairAssertions , that the subject changes etc

**Review by robstoll — CHANGES_REQUESTED** [review]

**Comment by BimuratMukhtar** [comment]
Do not look for now, i will add difference of two get function (fail fast behaviour)

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt (hunk 4 above)
Comment (BimuratMukhtar) [reply apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt:18]: ok

File: apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt (hunk 2 above)
Comment (BimuratMukhtar) [reply apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/ListAssertionSamples.kt:11]: ok

**Review by robstoll — CHANGES_REQUESTED** [review]

**Review by robstoll — APPROVED** [review]

**Comment by robstoll** [comment]
@BimuratMukhtar thanks for your second contribution :+1: 
How about a third one?

**Comment by BimuratMukhtar** [comment]
maybe later)

**Pull request merged**